Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,65 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './App.css';
+import { Button } from './components/ButtonComponents/NumberButton';
+import { Input } from './components/DisplayComponents/CalculatorDisplay';
+import { ClearButton } from './components/ButtonComponents/ActionButton';
+import * as math from 'mathjs';
 
-const App = () => {
-  return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
-  );
-};
+class App extends Component {
+  constructor(props) {
+    super(props);
 
-export default App;
+    this.state = {
+      input: ""
+    };
+  }
+
+  addToInput = val => {
+    this.setState({ input: this.state.input + val });
+  };
+
+  handleEqual = () => {
+    this.setState({input: math.eval(this.state.input)});
+  };
+
+  render() {
+    return (
+      <div className="app">
+        <div className="calc-wrapper">
+          <Input input={this.state.input} />
+          <div className="row">
+            <Button handleClick={this.addToInput}>7</Button>
+            <Button handleClick={this.addToInput}>8</Button>
+            <Button handleClick={this.addToInput}>9</Button>
+            <Button handleClick={this.addToInput}>÷</Button>
```

Comment:
This "÷" is not the JavsScript operator for division. Putting the division operator in there did the trick.

```suggestion
            <Button handleClick={this.addToInput}>/</Button>
```